x11-libs/xcb-util-image/xcb-util-image-0.4.0-r2.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+XORG_MULTILIB=yes
+inherit xorg-3
+
+DESCRIPTION="X C-language Bindings sample implementations"
+HOMEPAGE="https://xcb.freedesktop.org/ https://gitlab.freedesktop.org/xorg/lib/libxcb-image"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~sparc-solaris ~x64-solaris"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="x11-base/xorg-proto
```

Comment:
`xorg-proto` should be in `DEPEND`.

Reply:
Are you saying this one should be dropped from `RDEPEND`?  It has `DEPEND="${RDEPEND}"`.

Reply:
Exactly, let's move it from `RDEPEND` to `DEPEND`.